app/workers/zync_worker.rb

```diff
@@ -13,6 +13,15 @@ class ZyncWorker
 
   sidekiq_options queue: :zync
 
+  # It is used as the delay, in seconds.
+  # The default variable is the same as sidekiq's default, as you can see in https://github.com/mperham/sidekiq/wiki/Error-Handling#automatic-job-retry
+  # the default would last up 20 days and some hours, and we need it to last less than 7 days for the event to still exist,
+  # so it goes 3 times faster to last maximum 6 days and some hours (1/3 = 7/21)
+  sidekiq_retry_in do |count, _exception|
+    default = (count ** 4) + 15 + (rand(30) * (count + 1))
+    default / 3
```

Comment:
Remember that this will truncate to the integer part of the quotient.

Reply:
Yeah, that's fine 😄